.env.dev

```diff
@@ -1,9 +1,5 @@
-FLASK_APP=project/__init__.py
```

Comment:
Unfortunately this is still needed for Flask to understand where it's at, at least for commands such as `flask db upgrade` otherwise it's struggling to find a proper app. I specifically was having issues running the migration from a fresh install due to:
`Error: Could not locate a Flask application. You did not provide the "FLASK_APP" environment variable, and a "wsgi.py" or "app.py" module was not found in the current directory.`
without that env var set.